testing/gtest/APKBUILD

```diff
@@ -0,0 +1,55 @@
+# Maintainer: Francesco Colista <fcolista@alpinelinux.org>
```

Comment:
In fact, in Arch Linux, the gtest & gmock is combined into one PKGBUILD. But, I don't know how to do it proper in Alpine Linux.